doc/nrf/config_and_build/configuring_app/cmake/index.rst

```diff
@@ -115,6 +125,48 @@ This is how you can specify them:
 
 See :ref:`configuration_permanent_change` and Zephyr's :ref:`zephyr:west-building-cmake-args` for more information.
 
+Examples of commands
+--------------------
+
+Providing CMake variable for build types
+  .. toggle::
```

Comment:
```suggestion
Providing the CMake variable for build types:

  .. toggle::
```
Could also be "a CMake variable", I think. Or "CMake variables", not really sure.

Also, the toggle makes the tabs look off:
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/f93caa6e-2acd-4562-b19a-c867cff30d60)

Reply:
Let's see if this doesn't break the indented term-definition formatting.

Reply:
It does break it if that was the intention. I thought it might've been unintentional.